consumes.json

```diff
@@ -0,0 +1,352 @@
+    "System.Net.Security": "4.3.0-beta-24430-01",
+    "System.Net.Sockets": "4.3.0-beta-24430-01",
+    "System.Net.WebSockets": "4.3.0-beta-24430-01",
+    "System.Net.WebSockets.Client": "4.3.0-beta-24430-01",
+    "System.ObjectModel": "4.3.0-beta-24430-01",
+    "System.Reflection": "4.3.0-beta-24430-01",
+    "System.Reflection.Emit": "4.3.0-beta-24430-01",
+    "System.Reflection.Emit.ILGeneration": "4.3.0-beta-24430-01",
+    "System.Reflection.Emit.Lightweight": "4.3.0-beta-24430-01",
+    "System.Reflection.Extensions": "4.3.0-beta-24430-01",
+    "System.Reflection.Primitives": "4.3.0-beta-24430-01",
+    "System.Reflection.TypeExtensions": "4.3.0-beta-24430-01",
+    "System.Resources.ResourceManager": "4.3.0-beta-24430-01",
+    "System.Runtime": "4.3.0-beta-24430-01",
+    "System.Runtime.Extensions": "4.3.0-beta-24430-01",
+    "System.Runtime.Handles": "4.3.0-beta-24430-01",
+    "System.Runtime.InteropServices": "4.3.0-beta-24430-01",
+    "System.Runtime.InteropServices.RuntimeInformation": "4.3.0-beta-24430-01",
+    "System.Runtime.Loader": "4.3.0-beta-24430-01",
+    "System.Runtime.Numerics": "4.3.0-beta-24430-01",
+    "System.Runtime.Serialization.Json": "4.3.0-beta-24430-01",
+    "System.Runtime.Serialization.Xml": "4.3.0-beta-24430-01",
+    "System.Security.AccessControl": "4.3.0-beta-24430-01",
+    "System.Security.Claims": "4.3.0-beta-24430-01",
+    "System.Security.Cryptography.Algorithms": "4.3.0-beta-24430-01",
+    "System.Security.Cryptography.Cng": "4.3.0-beta-24430-01",
+    "System.Security.Cryptography.Encoding": "4.3.0-beta-24430-01",
+    "System.Security.Cryptography.Primitives": "4.3.0-beta-24430-01",
+    "System.Security.Cryptography.X509Certificates": "4.3.0-beta-24430-01",
+    "System.Security.Principal": "4.3.0-beta-24430-01",
+    "System.Security.Principal.Windows": "4.3.0-beta-24430-01",
+    "System.Text.Encoding": "4.3.0-beta-24430-01",
+    "System.Text.Encoding.CodePages": "4.3.0-beta-24430-01",
+    "System.Text.Encoding.Extensions": "4.3.0-beta-24430-01",
+    "System.Text.RegularExpressions": "4.3.0-beta-24430-01",
+    "System.Threading": "4.3.0-beta-24430-01",
+    "System.Threading.Overlapped": "4.3.0-beta-24430-01",
+    "System.Threading.Tasks": "4.3.0-beta-24430-01",
+    "System.Threading.Tasks.Parallel": "4.3.0-beta-24430-01",
+    "System.Threading.Thread": "4.3.0-beta-24430-01",
+    "System.Threading.ThreadPool": "4.3.0-beta-24430-01",
+    "System.Threading.Timer": "4.3.0-beta-24430-01",
+    "System.Xml.ReaderWriter": "4.3.0-beta-24430-01",
+    "System.Xml.XDocument": "4.3.0-beta-24430-01",
+    "System.Xml.XmlDocument": "4.3.0-beta-24430-01",
+    "System.Xml.XmlSerializer": "4.3.0-beta-24430-01",
+    "System.Xml.XPath": "4.3.0-beta-24430-01"
+  },
+  "toolset": {
+    "Microsoft.DotNet.BuildTools": "1.0.26-prerelease-00726-03",
+    "Microsoft.DotNet.BuildTools.RepoUtils": "1.0.0-prerelease-00726-03",
+    "test-runtime": null
```

Comment:
I think we can safely exclude `"target": "project"` dependencies like test-runtime since they aren't `consume`d as packages.